Test/client_test.py

```diff
@@ -38,6 +38,14 @@ def test_get_org(self):
         response = self.c.get_org("a_myOrg84")
         self.assertEqual("a_myOrg84",response['org_name'])
 
+    def test_get_environment(self):
+        client =Algorithmia.client(api_key=os.environ.get('ALGORITHMIA_API_KEY'))
+        response = client.get_environment("python2")
+        print(response)
+        if("error" in response):
+            print(response)
+        self.assertTrue(response is not None and "environments" in response)
```

Comment:
like the other PR; it would be great if we actually inspected the value in the response, but this isn't a blocker; we can address this in a follow-up.